addons/fleet/models/fleet_vehicle_log_contract.py

```diff
@@ -25,7 +25,10 @@ def compute_next_year_date(self, strdate):
     currency_id = fields.Many2one('res.currency', related='company_id.currency_id')
     name = fields.Char(string='Name', compute='_compute_contract_name', store=True, readonly=False)
     active = fields.Boolean(default=True)
-    user_id = fields.Many2one('res.users', 'Responsible', default=lambda self: self.env.user, index=True)
+    user_id = fields.Many2one('res.users',
+                              'Responsible',
+                              default=lambda self: self.env['fleet.vehicle'].browse(self._context.get('active_id')).manager_id,
+                              index=True)
```

Comment:
Let's be more consistent with the code below:
```suggestion
    user_id = fields.Many2one(
        comodel_name='res.users',
        string='Responsible',
        default=lambda self: self.env['fleet.vehicle'].browse(self._context.get('active_id')).manager_id,
        index=True)
```